src/ucp/core/ucp_ep.c

```diff
@@ -1155,6 +1184,16 @@ static void ucp_ep_discard_lanes_callback(void *request, ucs_status_t status,
         return;
     }
 
+    ucs_for_each_bit(lane, arg->unprogress_mask) {
```

Comment:
i think this code should be in ucp_worker_discard_uct_ep_destroy_progress, right before calling uct_ep_destory
maybe also unite code with ucp_ep_cleanup_lanes

Reply:
done. what i did includes:
1) pass `rsc_index` to `ucp_worker_discard_uct_ep`
2) pass `rsc_index` to `req->send.discard_uct_ep` (yes, i added a new field to `req->send.discard_uct_ep` so i might not pass CI because of `sizeof(req)` increased)
3) unprogress in `ucp_worker_discard_uct_ep_destroy_progress`
hope i understand correctly